src/components/html/EcAppCard.html

```diff
@@ -1,32 +1,46 @@
-<a-card class="ec-app-card" @click="$emit('click')" hoverable>
-  <div slot="cover" class="text-center">
-    <img
-      v-if="iconUrl"
-      class="img-fluid"
-      :src="iconUrl"
-      :alt="app.title"
+<div 
+  class="card" 
+  @click="$emit('click')"  
+  style="height: 26rem">
```

Comment:
```suggestion
>
```

Reply:
O card deve ser responsivo, aliás quase tudo deve ser sempre responsivo, fixando `height` ele deixa de ser.
Veja que no elemento anterior não tinha um `height` setado desta forma, não vejo razão pra fazer isto, se por acaso ele está ficando grande demais tem algum erro a ser corrigido, provavelmente de flexbox, em outro ponto e de outra forma, sem forçar altura fixa desta componente.